docker/hadoop/cdh/scripts/startup/51_hive_start

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -ex
+
+source /usr/sbin/get_hive_home.sh
```

Comment:
I think the `source` here is not necessary, the file is empty

Reply:
true, will remove